website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -0,0 +1,140 @@
+---
+layout: docs
+page_title: Upgrading to 1.10.0
+description: >-
+  Specific versions of Consul may have additional information about the upgrade
+  process beyond the standard flow.
+---
+
+# Upgrading to 1.10.0
+
+<EnterpriseAlert />
+
+## Introduction
+
+This guide explains how to best upgrade a single Consul Enterprise datacenter to v1.10.0
+from a version of Consul that is forwards compatible with v1.10. If you are on a major 
+version of Consul prior to 1.8, please follow the link for the version you are on from 
+[here](/docs/upgrading/instructions). If you are already on major versions 1.8 or 1.9
+this guide will go over the procedures required for upgrading to v1.10. If you have
+multiple Consul datacenters then they are upgraded in the normal order with each
+upgrade taking into account the instructions below.
+
+For the open source version of Consul please follow the 
+[General Upgrade Process](/docs/upgrading/instructions/general-process).
+
+~> You can only upgrade to Consul Enterprise 1.10 from a version of Consul Enterprise 
+1.8 >= 1.8.13 or 1.9 >= 1.9.7. Other versions of Consul Enterprise are not forwards
+compatible with v1.10 and will cause issues during the upgrade that could result in
+agents failing to start.
```

Comment:
```suggestion
compatible with v1.10 and will cause issues during the upgrade that could result in
client and snapshot agents failing to start due to [changes in the way we manage licenses](/??).
```
Not sure offhand where that link should go but maybe the FAQ on learn?

Reply:
Also it would be worth adding something like this to the callout box:

> This guide will walk through upgrading first to a compatible patch release and then completing the 1.10 upgrade from there.

Without that, I even was confused that right after saying "you need 1.8.13" the next paragraph below says " you need >= 1.8.0" before I realised why the discrepancy.

Reply:
I think I addressed both bits of this. I tried to make it clearer that this guide can really only be used if you are using versions 1.8+ and if you are not you should follow the normal upgrade procedures to get there first.